glib/src/param_spec.rs

```diff
@@ -1069,6 +1227,12 @@ impl ParamSpecBoxed {
     }
 }
 
+define_builder!(
+    ParamSpecBoxed,
+    ParamSpecBoxedBuilder,
+    default_value: crate::Type = crate::Type::BOXED,
```

Comment:
This is not the default value, it's the boxed type that this property can handle.

And it's mandatory and must be a concrete boxed type, so part of the `new()` / `builder()` function